src/collections/_documentation/workflow/alerts-notifications/alerts.md

```diff
@@ -0,0 +1,181 @@
+
+### Alert Details
+
+In this page, you can view a single instance of an alert from the time the alert is created to when the alert is resolved, or the current time if the alert is still active.
+
+[{% asset notifications/alert-details.png alt="Alert details illustrates a graph showing errors." %}]({% asset notifications/alert-details.png @path %})
+
+The "subscribe" button subscribes you to workflow notifications for the alert, which include status changes and comments.
+
+## Issue Alerts
+
+Issue alerts fire whenever any issue in the project matches the specified criteria, such as a resolved issue re-appearing or an issue affecting many users.
+
+### **Conditions**
+
+Conditions are evaluated for an issue alert **each time** the issue receives a new event, subject to [LINK: rate limits].
+
+**Each condition is evaluated independently of other conditions**. For example, the following alert will never fire:
+
+[{% asset notifications/alert-conditions2.png alt="The alert condition in this example shows the conditions are conflicting with each other." %}]({% asset notifications/alert-conditions2.png @path %})
+
+In the example, the alert will not fire because an event cannot satisfy both of these conditions simultaneously:
+
+- Sentry detects/creates a new issue
+- The issue has happened 10 times
+
+**Available Conditions**
+
+- Sentry detects/creates a new issue
+- An issue changes state from `resolved` to `unresolved`
+- An issue changes state from `ignored` to `unresolved`
+- An issue has happened more than {value} times in {interval}
+    - value: a positive integer
+    - interval: one minute, one hour, one day, one week, or 30 days
+- An issue has been seen by more than {value} users in {interval}
+    - value: a positive integer
+    - interval: one minute, one hour, one day, one week, or 30 days
+- Sentry detects a new event
+- Sentry detects a new event with tags matching {key} {match} {value}
+    - key: any tag [LINK: tags]
+    - match: equals, does not equal, starts with, ends with, contains, does not contain, is set, or is not set
+    - value: any key’s value
+- Sentry detects a new event with {attribute} {match} {value}
+    - attribute: `message`, `platform`, `environment`, `type`, `exception.type`, `exception.value`, `user.id`, `user.email`, `user.username`, `user.ip_address`, `http.method`, `http.url`, `stacktrace.code`, `stacktrace.module`, or `stacktrace.filename`
+    - match: equals, does not equal, starts with, ends with, contains, does not contain, is set, or is not set
+    - value: any attribute’s value
+- Sentry detects a new event with level {match} {level}
+    - match: equal to, less than or equal to, greater than or equal to
+    - level: fatal, error, warning, info, debug, or sample
+
+### **Actions**
+
+The following actions are available:
```

Comment:
FYI/heads up, this PR will change the wording: https://github.com/getsentry/sentry/pull/17644 not sure if this affects anything

Reply:
@adhiraj Will you confirm if @jeffkwoh 's PR affects our docs?

Reply:
It may, a little bit, but we don't know how long that will take so let's come back to it once his PR is merged